config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  resources :locations
```

Comment:
For whatever reason I feel more comfortable with devise at the top, possibly alphabetic comfort, habit, insanity.

Thoughts?